resources/public/api/conf/1.0/other_expenses_create_and_amend.yaml

```diff
@@ -0,0 +1,103 @@
+summary: Create and Amend Other Expenses
+description: |
+  This endpoint enables you to create and amend other expenses.
+  A National Insurance number and tax year must be provided.
+
+  ### Test data
+  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+  <table>
+      <thead>
+          <tr>
+              <th>Header Value (Gov-Test-Scenario)</th>
+              <th>Scenario</th>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td><p>N/A - DEFAULT</p></td>
+              <td><p>Simulate a success response.</p></td>
+          </tr>
+      </tbody>
+  </table>
+
+tags:
+  - Other Expenses
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+  - $ref: './common/headers.yaml#/components/parameters/contentTypeJson'
+  - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+security:
+  - User-Restricted:
+      - write:self-assessment
+
+requestBody:
+  content:
+    application/json:
+      schema:
+        $ref: './schemas/create_and_amend_other_expenses/request.json'
+      examples:
+        example-1:
+          description: Default Example
+          value:
+            $ref: './examples/create_and_amend_other_expenses/request.json'
+
+responses:
+  "200":
+    description: Success
+    headers:
+      X-CorrelationId:
+        $ref: './common/headers.yaml#/components/parameters/correlationId'
+    content:
+      application/json:
+        schema:
+          $ref: './schemas/create_and_amend_other_expenses/response.json'
+        examples:
+          example-response:
+            description: Example Response
+            value:
+              $ref: './examples/create_and_amend_other_expenses/response.json'
+
+  "400":
```

Comment:
Missing FORMAT_CUSTOMER_REFERENCE

Reply:
There's a note on this in https://hmrcdigital.slack.com/archives/C01RK0L464S/p1673627552163059?thread_ts=1673627542.441909&cid=C01RK0L464S - I've removed FORMAT_CUSTOMER_REFERENCE as it isn't in the technical spec or the code. Reqs spec needs updating